resources/js/Components/Form/EntryCreateForm.tsx

```diff
@@ -119,196 +115,143 @@ const EntryCreateForm = ({
     ]
 
     const stepOneContent = (
-        <>
-            <StyledTypography>
-                Fill out the following fields with your entry details.
-            </StyledTypography>
-            <Grid container>
-                <StyledGrid item xs={12} md={type === 'Edit' ? 12 : 5}>
-                    <TextField
-                        margin='normal'
-                        fullWidth
-                        id='adventure_id'
-                        select
-                        required
-                        label='Adventure Title'
-                        name='Adventure Title'
-                        SelectProps={{
-                            MenuProps: {PaperProps: {sx: {maxHeight: 300}}},
+        <Grid container spacing={2}>
+            <Grid item xs={12}>
+                <Typography>
+                    Fill out the following fields with your DM Entry details.
```

Comment:
This should be "Fill out the following fields with your entry details." and not DM entry details

Reply:
will fix this, thanks for pointing out